platform-ui-skin/src/main/webapp/skin/less/component/Sprites/Style.less

```diff
@@ -3651,6 +3651,26 @@ a:hover [class^="uiIconApp"], [class^="uiIconApp"]:hover,
   content: "\e6d5";
 }
 
+.fa-icon-members:before  {
+  content: "\f0c0";
+}
+
+.fa-icon-tasks:before  {
+  content: "\f0ae";
+}
+
+.fa-icon-notes:before  {
+  content: "\f328";
+}
+
+.fa-icon-SpaceWallet:before  {
+  content: "\f555";
+}
+
+.fa-icon-settings:before  {
+  content: "\f013";
+}
+
```

Comment:
```suggestion
```
Why redefining font-awesome classes ?

Reply:
In fact, by adding those classes, we will not be able anymore to upgrade the library font-awesome without impacting our source code (in straightforward way). By copying the content Id of icons, that can change when upgrading font-awesome library, we will have to make a full test of all our features each time we upgrade this library to ensure to use the corresponding content Id. This is a very bad practice that wasn't used before and please avoid adding icon classes this way. Thanks.